task/t2.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from utils.read_files import read_files
+
+
+file_values = read_files()
+
+Temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        Temperatures[file_value[0]] = float(file_value[1])
```

Comment:
you need to compare the min and max temp column values

Reply:
change the logic

Reply:
done